### PR TITLE
Create Video Card

### PR DESCRIPTION
**Pull request by ChuckBTaylor**
Inital Work on `VideoCard`. Also added some required dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@emailjs/browser": "^3.11.0",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
```
Comment (ChuckBTaylor) [inline package.json:8]: Required for MUI

File: src/assets/videos/videos.ts
Comment (ChuckBTaylor) [inline src/assets/videos/videos.ts:1]: This is where Kurt would update / add videos

File: src/components/atoms/headers/Header1.tsx
Comment (ChuckBTaylor) [inline src/components/atoms/headers/Header1.tsx:1]: Renamed `Header.tsx`

File: package.json (hunk 1 above)
Comment (kmdunn5) [reply package.json:8]: Checkout #13 for the styled component version of the MUI implementation

**Review by mwilkins74 — APPROVED** [review]
All of this looks great to me!
Even learned something new already - hadn't seen `useMemo()` before, but it makes perfect sense.

File: src/common/interfaces/IVideoCard.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Genre, Role } from "../constants/enums";
+
+// Should this interface live here? Do we want to have a separate file for this type of interface?
+export interface IVideoCard {
+  url: string; // The URL of the video
+  title: string; // The title of the video
+  clientName: string; // The name of the client
+  roles: Role[]; // The roles Kurt was for this video
+  genre: Genre; // The genre of the video
+  isRecentWork?: boolean; // If this video should be on the recent work section
```
Comment (kmdunn5) [inline src/common/interfaces/IVideoCard.ts:10]: How will Kurt order this? Just by whatever is first in the list? Just curious

**Review by kmdunn5 — APPROVED** [review]

**Pull request merged**